backend/functions/attorneys/attorneys.function.test.ts

```diff
@@ -21,33 +22,20 @@ describe('Attorneys Azure Function tests', () => {
     jest
       .spyOn(ContextCreator, 'getApplicationContextSession')
       .mockResolvedValue(MockData.getCamsSession());
+    jest.spyOn(AttorneyList.prototype, 'getAttorneyList').mockResolvedValue([]);
   });
 
-  it('Should call getAttorneyList with office id if parameter was passed in URL', async () => {
+  test('Should call getAttorneyList with office id if parameter was passed in URL', async () => {
     const attorneysListSpy = jest.spyOn(AttorneysController.prototype, 'getAttorneyList');
 
     await httpTrigger(context, request);
 
-    expect(attorneysListSpy).toHaveBeenCalledWith(expect.objectContaining({ officeId }));
+    expect(attorneysListSpy).toHaveBeenCalledWith(
+      expect.objectContaining({ query: { office_id: officeId } }),
+    );
   });
```

Comment:
Deleting this entire test due to no functionality for getting attorney list by office id.